scheduler/admin.py

```diff
@@ -179,3 +180,6 @@ class ShiftHelperAdmin(MembershipFilteredAdmin):
     list_display = ("id", "user_account", "shift", "joined_shift_at")
     list_filter = ("joined_shift_at",)
     raw_id_fields = ("user_account", "shift")
+
+
+admin.site.register(ShiftMessageToHelpers)
```

Comment:
ShiftMessageToHelpers should be "read-only" in admin area, as there seems to be no need to change any values.
It's admin values is "research only".